src/css/base.scss

```diff
@@ -1,77 +1,74 @@
-:root {
-  --accent-color: #f9bb47;
-  --background: #1a353f;
-  --shadow: #1a1b25;
-  --background-light: #015492;
-  --light-text: #bfc0c0;
-  --box-background: #ffffff;
-}
+@import 'variables';
+@import 'mixins';
 
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
-body {
+%flex_center-shared {
   align-items: center;
-  background-color: var(--background);
   display: flex;
+}
+
+body {
+  @extend %flex_center-shared;
+  background-color: $background_color;
   flex-direction: column;
   font-family: 'Noto Sans SC', sans-serif;
 }
 
 nav {
-  align-items: center;
+  @extend %flex_center-shared;
   background-image: linear-gradient(white, #e2e2e2);
   color: #2b8b9e;
-  display: flex;
-  font-size: 18px;
-  height: 50px;
+  font-size: $fontSize_18px;
+  height: 70px;
   justify-content: center;
   width: 100%;
+    h1 {
+      font-size: 3.3em;
```

Comment:
This is great, Allison!  It actually addresses one of our accessibility issues.  By changing the fonts to ems, this will allow for a screen reader.
I'll catch you up when I give you my lesson notes, but Khalid taught us a cool way where we can close issues directly from our PRs.

Reply:
Yes, let's also be consistent with our units! If we're going to switch to `ems`, let's try to do it everywhere possible rather than hardcoding some pixel sizes for our fonts